src/index.d.ts

```diff
@@ -208,16 +208,18 @@ declare namespace preact {
 		/** Attach a hook that is invoked after a vnode has rendered. */
 		diffed?(vnode: VNode): void;
 		/** Attach a hook that is invoked after an error is caught in a component but before calling lifecycle hooks */
-		catchError?(error: any, component: Component): void;
+		catchError?(error: any, vnode: VNode): void;
```

Comment:
wondering if there would be a use for vnodes here

Reply:
Honestly, I don't think there really is... I'm not super happy with this change. I have more thoughts about this. Will post more soon

Reply:
Personally, I don't mind the change for now. It could prove useful in the future for `preact/debug`. We could theoretically catch top-level errors and display a nice message in the console.

Reply:
#1692 contains where my brain went when making this change. Let me know what you think!

Reply:
Awesome, marking it as private is an even better solution :+1: